src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,8 +1,61 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final String SPACE = " ";
 
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+        LocalDate sdateFrom = LocalDate.parse(dateFrom, formatter);
+        LocalDate sdateTo = LocalDate.parse(dateTo, formatter);
+        int[] totalSalaryForPerson = new int[names.length];
+        for (int i = 0; i < data.length; i++) {
+            String[] dates = data[i].split(" ");
```

Comment:
use your constant please

```suggestion
            String[] info = data[i].split(SPACE);
```